R/toolEdgeTransportSA.R

```diff
@@ -47,7 +47,7 @@ toolEdgeTransportSA <- function(SSPscen,
   ########################################################
   if (is.null(outputFolder) & isStored) stop("Please provide an outputfolder to store your results")
   if (is.null(gdxPath)) {gdxPath <- file.path(getConfig("sourcefolder"),
-                                              "REMINDinputForTransportStandalone", "fulldata.gdx")}
```

Comment:
does this mean, you substituted the old fulldata.gdx file? Should we keep it to be able to reproduce old input data? Why not renaming the file which is read in?

Reply:
I use the approach we agreed on for inputdata versioning:
- create version folders and move the files to each folder
-  create a symlink in the root folder linking to the old file for backwards compatibility

Reply:
ah, it is a subfolder, fine!